templates/zerver/keyboard_shortcuts.html

```diff
@@ -76,6 +76,10 @@ <h3 id="keyboard-shortcuts-label">{% trans %}Keyboard shortcuts{% endtrans %}</h
           <td class="hotkey">Esc</td>
           <td class="definition">{% trans %}Cancel compose{% endtrans %}</td>
         </tr>
+        <tr>
+          <td class="hotkey">@</td>
+          <td class="definition">{% trans %}Compose a reply @-mentioning author{% endtrans %}</td>
+        </tr>
```

Comment:
This should also be documented in templates/zerver/help/keyboard_shortcuts.html.